src/ActiveLogin.Authentication.GrandId.Api/GrandIdDevelopmentApiClient.cs

```diff
@@ -43,6 +43,11 @@ public async Task<FederatedLoginResponse> FederatedLoginAsync(FederatedLoginRequ
         {
             await SimulateResponseDelay().ConfigureAwait(false);
 
+            if (!string.IsNullOrEmpty(request.PersonalIdentityNumber))
```

Comment:
This will cause concurrency issues as the dev implementation is registered as singleton so the _personalIdentityNumber would end up changing if someone logs in. Needs to stay readonly.

Reply:
True, completely forgot that this one is singleton. In that case we need to add such parameter to `SessionStateRequest` and populate it from `HandleRemoteAuthenticateAsync`, any other ideas?

Reply:
You can just add it to the list of federatedRequests. Just need to slightly modify what that contains. I can show later when I have access to a computer.